.coveragerc

```diff
@@ -7,3 +7,7 @@ omit =
     */sklearn/_build_utils/*
     */benchmarks/*
     **/setup.py
+
+[report]
+exclude_lines =
+    pragma: no cover
```

Comment:
Main idea behind it is to make a second run of this PR in the future much faster (which involves opening all modules that show >0 uncovered lines in the coverage report, which are quite a few). Granted, this may be too little benefit to justify this change.

Reply:
OK, I have mixed feeling about it. Let's leave `pragma: no cover` for now as is, and see what the second reviewer think about it.